resources/views/partials/forms/edit/status-select.blade.php

```diff
@@ -4,15 +4,12 @@
     {{ Form::label($fieldname, $translated_name, array('class' => 'col-md-3 control-label')) }}
 
     <div class="col-md-7{{  ((isset($required)) && ($required=='true')) ? ' required' : '' }}">
-        <select class="js-data-ajax" data-endpoint="statuslabels" data-placeholder="{{ trans('general.select_statuslabel') }}" name="{{ $fieldname }}" style="width: 100%" id="status_select_id" aria-label="{{ $fieldname }}" {!!  ((isset($item)) && (Helper::checkIfRequired($item, $fieldname))) ? ' required ' : '' !!}{{ (isset($multiple) && ($multiple=='true')) ? " multiple='multiple'" : '' }}>
-            @if ($status_id = old($fieldname,  (isset($item)) ? $item->{$fieldname} : ''))
-                <option value="{{ $status_id }}" selected="selected" role="option" aria-selected="true"  role="option">
-                    {{ (\App\Models\Statuslabel::find($status_id)) ? \App\Models\Statuslabel::find($status_id)->name : '' }}
-                </option>
-            @else
-                <option value=""  role="option">{{ trans('general.select_status') }}</option>
```

Comment:
IIRC, we have that here because in some places these fields are required and in some they are not, and the dropdowns don't provide a "Select blah" option automatically. This would need to be tested across other things that use these dropdown partials.